gptokeyb.cpp

```diff
@@ -479,9 +480,9 @@ void readConfigFile(const char* config_file)
     } else if (strcmp(co.key, "deadzone_triggers") == 0) {
       config.deadzone_triggers = atoi(co.value);
     } else if (strcmp(co.key, "mouse_scale") == 0) {
-      FAKE_MOUSE_SCALE = atoi(co.value);
-    } else if (strcmp(co.key, "mouse_speed") == 0) {
-      FAKE_MOUSE_SPEED = atoi(co.value);
+      fake_mouse_scale = atoi(co.value);
```

Comment:
This needs to be `config.fake_mouse_scale` now - same for the delay and all other places where the values are used.

Reply:
Done, I missed that part